recipes-core/images/includes/nilrt-image-base.inc

```diff
@@ -14,6 +24,13 @@ IMAGE_INSTALL += "\
 	packagegroup-ni-tzdata \
 "
 
+# Add the package sources needed when building the SDK
+IPK_FEED_URIS:append:task-populate-sdk = "\
+    NIOE-dummy-sdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-target \
+    NIOE-nativesdk##${NILRT_LOCAL_FEED_URI}/${SDK_ARCH}-${SDKPKGSUFFIX} \
+    NIOE-dummy-nativesdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-${SDKPKGSUFFIX} \
+"
```

Comment:
This only needs to apply to SDK tasks as packages in these feeds PROVIDE packages that should be installed normally elsewhere. I didn't see any issues always including them, but this feels "cleaner".

Reply:
It's not currently, but I could also make this conditional like in meta-toolchain.bbappend.